spirv_glsl.cpp

```diff
@@ -778,16 +772,28 @@ void CompilerGLSL::emit_header()
 		{
 			if (options.version < 310)
 				SPIRV_CROSS_THROW("At least ESSL 3.10 required for fragment shader interlock.");
-			require_extension_internal("GL_NV_fragment_shader_interlock");
+			statement("#extension GL_NV_fragment_shader_interlock : enable");
+			statement("#define SPIRV_Cross_beginInvocationInterlock() beginInvocationInterlockNV()");
+			statement("#define SPIRV_Cross_endInvocationInterlock() endInvocationInterlockNV()");
 		}
 		else
 		{
 			if (options.version < 420)
-				require_extension_internal("GL_ARB_shader_image_load_store");
-			require_extension_internal("GL_ARB_fragment_shader_interlock");
+				statement("#extension GL_ARB_shader_image_load_store : enable");
+
+		    statement("#ifdef GL_ARB_fragment_shader_interlock");
```

Comment:
These ifdef ladders can be handled in the code that emits extensions below. I'd prefer consistency in this case even if it's not strictly necessary for correctness.